devhelp/AUTHORS

```diff
@@ -0,0 +1 @@
+Matthew Brush <mbrush@codebrainz.ca>
```

Comment:
Maybe add yourself here?

Reply:
I changed it, but Matthew Brush who created the plugin.
I just updated to gtk+3 version